pom.xml

```diff
@@ -33,6 +34,21 @@
             <version>${aws.version}</version>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Maybe not, but from my experience, really weird things happen if the versions are out of sync.